cross/ddns-go/Makefile

```
@@ -0,0 +1,33 @@
+PKG_NAME = ddns-go
+PKG_VERS = 6.12.0
+WEB_VERSION = $(PKG_VERS)
+PKG_EXT = tar.gz
+PKG_DIST_NAME = v$(PKG_VERS).$(PKG_EXT)
+PKG_DIST_SITE = https://github.com/jeessy2/ddns-go/archive
+PKG_DIST_FILE = $(PKG_NAME)-$(PKG_VERS).$(PKG_EXT)
+PKG_DIR = $(PKG_NAME)-$(PKG_VERS)
+
+BUILD_DEPENDS = native/go
+
+# lacking support for atomics
+UNSUPPORTED_ARCHS = $(ARMv5_ARCHS)
+# not supported by go
+UNSUPPORTED_ARCHS += $(PPC_ARCHS)
+
+HOMEPAGE = https://github.com/jeessy2/ddns-go
+COMMENT  = Automatically obtain your public IPv4 or IPv6 address and resolve it to the corresponding domain name service.
+LICENSE = MIT
+
+COMPILE_TARGET = ddns-go_compile_target
+INSTALL_TARGET = ddns-go_install
+
+include ../../mk/spksrc.cross-go.mk
+
+.PHONY: ddns-go_compile_target
+ddns-go_compile_target:
+	@$(RUN) $(MAKE) VERSION=$(shell git describe --tags `git rev-list --tags --max-count=1` 2>/dev/null || git rev-parse --short HEAD) BUILD_TIME="$$(date -u +'%Y-%m-%dT%H:%M:%SZ')" build
```

We can't take the version from git, because we build from source within the git repository of spksrc.
currently the version looks like this:
```
./ddns-go -v
b8362bd
```

An official release shows
```
 ./ddns-go -v
v6.12.0
```

We could have the same with:

```suggestion
ddns-go_compile_target:
	@$(RUN) $(MAKE) VERSION=v$(PKG_VERS) BUILD_TIME="$$(date -u +'%Y-%m-%dT%H:%M:%SZ')" build
```